### ISSUE TITLE
hello evryone!!when I  python/train/train.py ,meet this issue :Restored 267(544) vars from /home/bc/work/fastMaskCnn/data/pretrained_models/resnet_v1_50.ckpt 2017-09-11 19:56:08.315092: W tensorflow/core/framework/op_kernel.cc:1192] Invalid argument: Size 0 must be non-negative, not -1113667835 2017-09-11 19:56:08.315156: F tensorflow/core/framework/tensor_shape.cc:249] Check failed: 0 <= new_num_elements (0 vs. -1) 2017-09-11 19:56:08.315190: W tensorflow/core/framework/op_kernel.cc:1192] Invalid argument: Input to reshape is a tensor with 55 values, but the requested shape has 5120793180 [[Node: Reshape = Reshape[T=DT_FLOAT, Tshape=DT_INT32, _device="/job:localhost/replica:0/task:0/device:GPU:0"](DecodeRaw_1/_559, Reshape/shape)]] Aborted (core dumped)  if anyone has idea,please tell me

### ISSUE THREAD

**Issue by zhanglijian** [CharlesShang/FastMaskRCNN#144]
restoring  resnet_v1_50/block4/unit_3/bottleneck_v1/conv1/BatchNorm/gamma:0
restoring  resnet_v1_50/block4/unit_3/bottleneck_v1/conv1/BatchNorm/beta:0
restoring  resnet_v1_50/block4/unit_3/bottleneck_v1/conv1/BatchNorm/moving_mean:0
restoring  resnet_v1_50/block4/unit_3/bottleneck_v1/conv1/BatchNorm/moving_variance:0
restoring  resnet_v1_50/block4/unit_3/bottleneck_v1/conv2/weights:0
restoring  resnet_v1_50/block4/unit_3/bottleneck_v1/conv2/BatchNorm/gamma:0
restoring  resnet_v1_50/block4/unit_3/bottleneck_v1/conv2/BatchNorm/beta:0
restoring  resnet_v1_50/block4/unit_3/bottleneck_v1/conv2/BatchNorm/moving_mean:0
restoring  resnet_v1_50/block4/unit_3/bottleneck_v1/conv2/BatchNorm/moving_variance:0
restoring  resnet_v1_50/block4/unit_3/bottleneck_v1/conv3/weights:0
restoring  resnet_v1_50/block4/unit_3/bottleneck_v1/conv3/BatchNorm/gamma:0
restoring  resnet_v1_50/block4/unit_3/bottleneck_v1/conv3/BatchNorm/beta:0
restoring  resnet_v1_50/block4/unit_3/bottleneck_v1/conv3/BatchNorm/moving_mean:0
restoring  resnet_v1_50/block4/unit_3/bottleneck_v1/conv3/BatchNorm/moving_variance:0
restoring  resnet_v1_50/logits/weights:0
restoring  resnet_v1_50/logits/biases:0
Restored 267(544) vars from /home/bc/work/fastMaskCnn/data/pretrained_models/resnet_v1_50.ckpt
2017-09-11 19:56:08.315092: W tensorflow/core/framework/op_kernel.cc:1192] Invalid argument: Size 0 must be non-negative, not -1113667835
2017-09-11 19:56:08.315156: F tensorflow/core/framework/tensor_shape.cc:249] Check failed: 0 <= new_num_elements (0 vs. -1)
2017-09-11 19:56:08.315190: W tensorflow/core/framework/op_kernel.cc:1192] Invalid argument: Input to reshape is a tensor with 55 values, but the requested shape has 5120793180
	 [[Node: Reshape = Reshape[T=DT_FLOAT, Tshape=DT_INT32, _device="/job:localhost/replica:0/task:0/device:GPU:0"](DecodeRaw_1/_559, Reshape/shape)]]
Aborted (core dumped)

**Comment by w-tingting** [comment]
I have the same problem,how do you solve it?

**Comment by sebastianbk** [comment]
I also have this issue. Is there any solution?
